optuna/storages/base.py

```diff
@@ -194,29 +532,73 @@ def get_best_trial(self, study_id):
 
         return copy.deepcopy(best_trial)
 
-    def get_trial_params(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_params(self, trial_id: int) -> Dict[str, Any]:
+        """Read parameter dictionary of a specified trial.
+
+        Args:
+            trial_id:
+                A trial ID of a trial to read parameters.
 
+        Returns:
+            A dictionary of a parameters consisting of parameter names and internal
+            representations of the parameters' values.
+
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching ``trial_id`` exists.
+        """
         return self.get_trial(trial_id).params
 
-    def get_trial_user_attrs(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_user_attrs(self, trial_id: int) -> Dict[str, Any]:
+        """Read a user-defined attributes of a specified trial.
+
+        Args:
+            trial_id:
+                A trial ID of a trial to read user-defined attributes.
 
+        Returns:
+            A dictionary of user-defined attributes of the trial.
+
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching ``trial_id`` exists.
+        """
         return self.get_trial(trial_id).user_attrs
 
-    def get_trial_system_attrs(self, trial_id):
-        # type: (int) -> Dict[str, Any]
+    def get_trial_system_attrs(self, trial_id: int) -> Dict[str, Any]:
+        """Read an optuna-internal attributes of a specified trial.
 
-        return self.get_trial(trial_id).system_attrs
+        Args:
+            trial_id:
+                A trial ID of a trial to read optuna-internal attributes.
 
-    def remove_session(self):
-        # type: () -> None
+        Returns:
+            A dictionary of optuna-internal attributes of the trial.
 
+        Raises:
+            :exc:`KeyError`:
+                If no trial with the matching ``trial_id`` exists.
+        """
+        return self.get_trial(trial_id).system_attrs
+
+    def remove_session(self) -> None:
+        """Clean up all connections to a database."""
         pass
 
-    def check_trial_is_updatable(self, trial_id, trial_state):
-        # type: (int, TrialState) -> None
+    def check_trial_is_updatable(self, trial_id: int, trial_state: TrialState) -> None:
+        """Check whether a `trial_state` is updatable.
```

Comment:
How about two backticks, or as plain text
```suggestion
        """Check whether a trial state is updatable.
```